### PR TITLE
git push -u origin master

### PR DESCRIPTION
**Pull request by DaemonSpade**

### CODE REVIEW

File: supported_servers.json [hunk 1]
```diff
@@ -745,6 +745,11 @@
     "site": "https://landverse.com/",
     "hpAddress": "0x012BE4CC",
     "nameAddress": "0x012C0F08"
+  },
+  {
+    "name": "uMMO",
+    "hpAddress": "010DCE10",
+    "nameAddress": "010D5DA8"
   }
```
[the remark this answers is not in the document]
Comment (afonsocdaniel) [reply supported_servers.json:754]: Hey @DaemonSpade, thanks for the PR. Do you mind to add the site of this server?

**Comment by DaemonSpade** [comment]
Right now we don't have a website I will update you once we already have
one this is a fresh server.

File: supported_servers.json (hunk 1 above)
[the remark this answers is not in the document]
Comment (DaemonSpade) [reply supported_servers.json:754]: Any updates on this?

[the remark this answers is not in the document]
Comment (DaemonSpade) [reply supported_servers.json:754]: Thank you

**Pull request merged**